index.bs

```diff
@@ -214,45 +215,56 @@ In other words, this attribute returns [=latest reading=]["quaternion"].
 
 ### OrientationSensor.populateMatrix() ### {#orientationsensor-populatematrix}
 
-The {{OrientationSensor/populateMatrix()}} method  populates the given buffer with WebGL-compatible rotation matrix
+The {{OrientationSensor/populateMatrix()}} method  populates the given object with rotation matrix
 which is converted from the value of [=latest reading=]["quaternion"] [[QUATCONV]], as shown below
 
 <img src="quaternion_to_rotation_matrix.png" style="display: block;margin: auto;" alt="Converting quaternion to rotation matrix." width="50%" height="50%">
 
 where:
 
- - W = cos(θ/2).
- - X = V<sub>x</sub> * sin(θ/2).
- - Y = V<sub>y</sub> * sin(θ/2).
- - Z = V<sub>z</sub> * sin(θ/2).
+ - W = cos(θ/2)
+ - X = V<sub>x</sub> * sin(θ/2)
+ - Y = V<sub>y</sub> * sin(θ/2)
+ - Z = V<sub>z</sub> * sin(θ/2)
 
-The rotation matrix is flattened in |targetBuffer| according to the column-major order.
+The rotation matrix is flattened in |targetMatrix| according to the column-major order.
```

Comment:
Yes, rotation matrix is calculated from quat as shown in the illustration and flattened into array or m11-m44 fields. So, I think this sentence is valid.